Functions.Templates/Templates/SqlOutputBinding-PowerShell/run.ps1

```diff
@@ -0,0 +1,28 @@
+<#
+Sample SQL Output Binding
+See https://aka.ms/sqlbindingsinput for more information about using this binding
+These tasks should be completed prior to running :
+    1. Update "commandText" in function.json - this should be the name of the table that you wish to upsert values to
+    2. Add an app setting named "SqlConnectionString" containing the connection string to use for the SQL connection
+    3. Change the bundle name in host.json to "Microsoft.Azure.Functions.ExtensionBundle.Preview" and the version to "[3.*, 4.0.0)"
+@param Request The HttpRequest that triggered this function
+#>
+using namespace System.Net
+
+# Trigger binding data passed in via param block
+param($Request)
+
+# Write to the Azure Functions log stream.
+Write-Host "PowerShell SQL Binding function processed a request."
+
+# Set results array to req_body for upsertion
```

Comment:
How is this being associated? The binding defines the name as `results` - which in the other languages is the name of the variable that you assign a value to. 

```json
      {
        "name": "results",
        "type": "sql",
        "direction": "out",
        "commandText": "[dbo].[table1]",
        "commandType": "Text",
        "connectionStringSetting": "SQLConnectionString"
      }
```

For example here's how JS does it : 

`    context.bindings.results = JSON.stringify(req.body);`

Reply:
Do we need something like this: `Push-OutputBinding -Name results -Value $req_body`?
Similar to Kafka Output Binding example: https://github.com/Azure/azure-functions-templates/blob/104122583f35c42a383b5d06ce5497932bfad636/Functions.Templates/Templates/KafkaOutput-PowerShell/run.ps1#L21